okhttp/src/test/java/okhttp3/internal/http2/Http2Test.java

```diff
@@ -646,6 +646,39 @@ public void pushPromise(int streamId, int promisedStreamId, List<Header> headerB
     });
   }
 
+  @Test public void ignoresEarlyHints() throws IOException {
```

Comment:
I would have expected a regular MockWebServer test, not an HTTP/2 implementation detail test. Is there something this approach makes better?